obal/data/roles/repoclosure/tasks/regular.yml

```diff
@@ -0,0 +1,17 @@
+- name: 'Run repoclosure'
```

Comment:
Remind me what makes it regular vs. irregular?

Reply:
`irregular` = `dist` = "we downloaded a set of RPMs and test them WITH a set of repos"
`regular` = "here are the repos, run repoclosure"

Totally taking better names (or, more probably, will try to make those two files one).

Reply:
I made it one now